predictive_test_selection/collect_pts_data.py

```diff
@@ -0,0 +1,93 @@
+import os
+import subprocess
+from datetime import datetime
+from enum import IntFlag, auto
+from typing import Any, List, Mapping
+
+# Eveything below is work in progress for part 2 when we already have historic data for the failure rates.
```

Comment:
This file is not being used at the moment but will be used during phase 2 of the data collection